### PR TITLE
Create actions.md

### PR DESCRIPTION
**Pull request by lucca65**
Add a little explanation on the claim process

### CODE REVIEW

File: features/actions.md
Comment (victorgdev) [inline features/actions.md]: @lucca65 assuming this documentation is for onboarding purposes (partners and team). 

Here is my suggestion for this section:
_Quantity_ :  Determines the number of times an action can be **Approved or Disapproved**. Users can only claim an action and be rewarded the amount of times specified in quantity. An example is: an action that have `Quantity of 3` votes; This action can have 100s of opened claims, but after the first 3 votes (approved or disapproved) all of the opened claims, not voted yet, will become "unvoteable", and the action status will change to "completed".

Question:
- Claims that have "n" votes, independently of the votes direction (approved or disapproved), if "quantity = n" the status of the action changes to "completed", correct?

File: features/actions.md
Comment (victorgdev) [inline features/actions.md]: Here is my suggestion for this section: 
- _Date_: An action can be claimed until certain date is reached. After the set date, claims cannot be opened. The validation of opened claims can occur after the set date. The opening of new claims cannot occur after the set date.

File: features/actions.md
Comment (victorgdev) [inline features/actions.md]: Here is my suggestion for this section:
Verification is the process by which the community decides if an action is completed or not. The verification process can be *Automatic* if validated by software, or *Manual*, which requires human validation.

File: features/actions.md
Comment (lucca65) [reply features/actions.md]: That's what I was trying to clarify on that part:

> That means that the users can claim all they want, but only the specified number can lead to rewards.

About your question:

>  n" the status of the action changes to "completed", correct?

So its not about **votes**. Its about **claims**. 

Lets say you create an action called "Plant a tree", that belongs to an objective called "Plant 2 trees". In this case, we configure that action to have `quantity` as 2. That is, after the community has verified that exactly 2 trees were planted, the action will be considered completed. 

Lets say that this community had high engagement, and a lot of users have claimed to plant trees. Lets assume that we had 10 claims open. 

Now the verifiers of that **action** will have check and give their vote on each **claim**. Lets say that they disapprove the first, approved the second. When its time to vote for the third one, their vote will fail. The action be considered completed.

All those claims will be unvotable

File: features/actions.md
Comment (lucca65) [reply features/actions.md]: What you think of removing the repetition?

```suggestion
  - _Date_: An action can be claimed until certain date is reached. After the set date, claims cannot be opened. The validation of opened claims can occur after the set date.
```

File: features/actions.md
Comment (lucca65) [reply features/actions.md]: Ok!

File: features/actions.md
Comment (victorgdev) [reply features/actions.md]: Good idea!

File: features/actions.md
Comment (lucca65) [reply features/actions.md]: I've updated this section @victorgdev let me know what you think

**Review by victorgdev — APPROVED** [review]
Awesome!

**Pull request merged**
